docs/rules/operation-4xx-problem-details-rfc7807.md

```diff
@@ -0,0 +1,84 @@
+# operation-4xx-problem-details-rfc7807
+
+Ensures that every operation with (400-499) HTTP response has content-type `application/problem+json` and fields `title` and `type`.
+
+| OAS | Compatibility |
+| --- | ------------- |
+| 2.0 | ❌            |
+| 3.0 | ✅            |
+| 3.1 | ✅            |
+
+## API design principles
+
+Problem Details for HTTP APIs as a way to carry machine-
+readable details of errors in a HTTP response to avoid the need to
+define new error response formats for HTTP APIs.
+
+Every operation with (400-499) HTTP response should have content-type `application/problem+json` and fields `title` and `type`. According to [specification](https://datatracker.ietf.org/doc/html/rfc7807).
+
+## Configuration
+
+| Option   | Type   | Description                                                                               |
+| -------- | ------ | ----------------------------------------------------------------------------------------- |
+| severity | string | Possible values: `off`, `warn`, `error`. Default `warn` (in `recommended` configuration). |
```

Comment:
Hmmm, I don't think we should warn on this one in the recommended. I think it should be `off` in the recommended config.

Reply:
I will change this. Please, review all document